src/Personal/utils.js

```diff
@@ -0,0 +1,25 @@
+function emailValidation(email) {
+  const validateEmail = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return validateEmail.test(String(email).toLowerCase());
+}
+
+function passwordValidation(password) {
+  const validatePassword = /^[0-9a-zA-Z]{8,16}$/;
```

Comment:
support special characters?

Reply:
fixed. 
now at least one special character should be used in password.